src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Avatar
+description: >
+  Avatars display typography, icons, or images in different sizes. They help users quickly recognize
+  an entity.
+package: '@zendeskgarden/react-avatars'
+propSheets:
+  - avatars/src/elements/Avatar.tsx
+---
+
+<Usage>
+  <div>
+    <MD isBold>When to use</MD>
+    <ul type="disc">
+      <li>To briefly explain the function of a user interface element</li>
+      <li>To give contextual help or information regarding ambiguous actions</li>
+    </ul>
+  </div>
+  <div>
+    <MD isBold>When NOT to use</MD>
+    <ul type="disc">
+      <li>To display actions vital to using the UI</li>
+      <li>To display truncated text a native `title` attribute should be used</li>
+    </ul>
+  </div>
+</Usage>
+
+## Examples
+
+### Shape
+
+An Avatar can be a circle or rounded square. View the shape usage best practices.
+
+import AvatarShape from '../../examples/avatar/AvatarShape.tsx';
+import AvatarShapeCode from '!!raw-loader!../../examples/avatar/AvatarShape.tsx';
+
+<CodeExample code={AvatarShapeCode}>
+  <AvatarShape />
+</CodeExample>
+
+### Size
+
+Sizing can be adjusted to displayed sizes. The default size of an Avatar is `medium`.
+
+import AvatarSize from '../../examples/avatar/AvatarSize.tsx';
+import AvatarSizeCode from '!!raw-loader!../../examples/avatar/AvatarSize.tsx';
+
+<CodeExample code={AvatarSizeCode}>
+  <AvatarSize />
+</CodeExample>
+
+### Type
+
+An Avatar can contain an icon, image, or text.
+
+import AvatarType from '../../examples/avatar/AvatarType.tsx';
+import AvatarTypeCode from '!!raw-loader!../../examples/avatar/AvatarType.tsx';
+
+<CodeExample code={AvatarTypeCode}>
+  <AvatarType />
+</CodeExample>
+
+### Status
+
+A status ring can be applied to the outside of an Avatar to communicate a user is away, available,
+or active.
+
+import AvatarStatus from '../../examples/avatar/AvatarStatus.tsx';
+import AvatarStatusCode from '!!raw-loader!../../examples/avatar/AvatarStatus.tsx';
+
+<CodeExample code={AvatarStatusCode}>
+  <AvatarStatus />
+</CodeExample>
+
+## Best practices
+
+### Selecting the right Avatar shape
+
+Circular Avatars are used to represent a person because the shape is most often associated with a
+profile picture. Rounded squares are used for system applications like objects, brands, or products.
+This helps the user quickly distinguish between a person or a system when scanning.
+
+<BestPractice>
+  <BestPracticeSection type="do" imageSource="components-avatar-shape-circle">
+    <ul>
+      <li>Use a circular shape to represent a person.</li>
+    </ul>
+  </BestPracticeSection>
+  <BestPracticeSection type="dont" imageSource="components-avatar-shape-square">
+    <ul>
+      <li>Use the rounded square shape to represent objects, brands, or products</li>
+    </ul>
+  </BestPracticeSection>
+</BestPractice>
+
+## Package
+
+<PackageDescription data={props.data.mdx.package} />
+<PropSheets data={props.data.mdx.propsSheets} reactPackage={props.data.mdx.package} />
```

Comment:
Should we have an `h2` saying `API` or `Prop Sheets` so the prop sheets are more identifiable from the side-bar?

Reply:
I used `Button` component page as a guide so didn't include "API" or "Prop Sheets". This is a good idea though, and I will make sure to update this accordingly in both component pages.

Reply:
@rossmoody do you think this would be worthwhile for all the component pages?

Reply:
Paired with @rossmoody on this, and we think so! I'm going to leave this off (for now). Ross is working on updating the heading hierarchy on a separate PR. Once Ross has outlined the heading structure, I'll update this 💯 .